### PR TITLE
Add Creative Commons Attribution-ShareAlike 4.0 International Public

### LINKED ISSUE #12: select license for content
maybe https://creativecommons.org/licenses/by-sa/4.0/

and add this to the footer

### PR DESCRIPTION
**Pull request by vyp**
License

Closes #12.

---

Disclaimer: IANAL so I don't know if this is the correct way to do it.

I guess the license should be added early before more contributions come in? Not sure if you want the full text file in the repo, or just a link to the license page in the readme footer? Also whether or not you want the "in short" description. If you don't want them, let me know and I'll remove them!

### CODE REVIEW

**Comment by garbas** [comment]
:+1: 

@vyp can you also add a link in foot and point it to https://creativecommons.org/licenses/by-sa/4.0/?

**Comment by vyp** [comment]
Hmm that link is already there in the first paragraph/line under `## License`, click "Display the rich diff" to see what I mean. Maybe instead of that you want a 'more explicit' link showing the url itself?

**Comment by garbas** [comment]
@vyp oh i ment in the footer of every generated page in `default.nix`

**Comment by vyp** [comment]
Makes sense now. Will do!

File: default.nix [hunk 1]
```diff
@@ -138,6 +138,7 @@ let
                   <ul class="list-unstyled">
                     <li><a href="https://nixos.org">NixOS</a></li>
                     <li><a href="https://github.com/nixos/nixos-weekly">Source Code</a></li>
+                    <li><a href="https://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA</a></li>
```
Comment (vyp) [inline default.nix:195]: @garbas Sorry for the delay, did you mean something like this?

**Pull request merged**